tests/conftest.py

```diff
@@ -35,3 +39,24 @@ def _get_fn(fn):
         return f"{test_dir}/data/{fn}"
 
     return _get_fn
+
+
+@pytest.fixture
+def gen_random_ptraj(request, tmp_path):
```

Comment:
Can `tmp_path` be removed here? I don't see that it's used (the tests pass locally for me without it).

If you remove `tmp_path`, you could set the scope of the fixture to `class`, but this really isn't critical.